srml/contracts/src/tests.rs

```diff
@@ -1256,17 +1256,24 @@ fn default_rent_allowance_on_create() {
 const CODE_RESTORATION: &str = r#"
 (module
 	(import "env" "ext_set_storage" (func $ext_set_storage (param i32 i32 i32 i32)))
-	(import "env" "ext_dispatch_call" (func $ext_dispatch_call (param i32 i32)))
+	(import "env" "ext_restore_to" (func $ext_restore_to (param i32 i32 i32 i32 i32 i32 i32 i32)))
 	(import "env" "memory" (memory 1 1))
 
 	(func (export "call")
-		(call $ext_dispatch_call
-			;; Pointer to the start of the encoded call buffer
-			(i32.const 200)
-			;; The length of the encoded call buffer.
-			;;
-			;; NB: This is required to keep in sync with the values in `restoration`.
-			(i32.const 115)
+		(call $ext_restore_to
+			;; Pointer and length of the encoded dest buffer.
+			(i32.const 256)
+			(i32.const 8)
+			;; Pointer and length of the encoded code hash buffer
+			(i32.const 264)
+			(i32.const 32)
+			;; Pointer and length of the encoded rent_allowance buffer
+			(i32.const 296)
+			(i32.const 8)
+			;; Pointer and number of items in the delta buffer.
+			;; This buffer specifies multiple keys for removal before restoration.
+			(i32.const 100)
+			(i32.const 1)
```

Comment:
so you remove the test for if the same key is given two times, but fine.

Reply:
Yeah, I realized that it won't be so nice : )